Core/Systems/Affixes/ItemTypes/MovementSpeedAffixes.cs

```diff
@@ -1,9 +1,9 @@
-﻿namespace PathOfTerraria.Core.Systems.Affixes.ItemTypes.ArmorAffixes;
+﻿namespace PathOfTerraria.Core.Systems.Affixes.ItemTypes;
 
 internal class MovementSpeed : ItemAffix
 {
 	public override void ApplyAffix(EntityModifier modifier, PoTItem item)
 	{
-		modifier.MovementSpeed += Value;
+		modifier.MovementSpeed += Value / 10;
```

Comment:
Same question, why /10 here?

Reply:
With a tier of 
```json
			{
				"minValue": 1,
				"maxValue": 4,
				"minimumLevel": 1,
				"weight": 1
			},
```
I was seeing the rolling of the item having 10-40% movespeed. This was to match the actual number in the json.

Reply:
1 = +100%

I assume what you were looking at was the implicit but that is constant, (4*100/10 =) +40% movement speed seems a little much on ilevel 10 gear
![billede](https://github.com/Path-of-Terraria/PathOfTerraria/assets/65250596/82526fcb-d40a-4680-8933-ab8b8d406c40)

Reply:
We will need to make it so that a affix can only roll a single time on an item. But lets make that a future update xD

Reply:
For the values themselves. Tier 1 is meant to be 1-4% as an affix. Shouldn't be rolling 15.28%

Reply:
You don't sound like you acknowledge what i said...

But that is probably true (still unrelated to the point though)

Reply:
> For the values themselves. Tier 1 is meant to be 1-4% as an affix. Shouldn't be rolling 15.28%

Dividing by 10 makes it roll 10-40

Reply:
If the boots roll 8% -> The value coming in is 0.8 already

![image](https://github.com/Path-of-Terraria/PathOfTerraria/assets/24656209/199a355d-ba16-4fa5-8644-217e3049fe6d)

So dividing it by 10 was making it 0.08

Reply:
What item is this on?

Reply:
because
> I assume what you were looking at was the implicit but that is constant

i still think this is the case...

Reply:
it dosent get rolled at all, and its 0.8, so like...
![billede](https://github.com/Path-of-Terraria/PathOfTerraria/assets/65250596/f78f0244-d89c-45bf-aa04-573862352b1c)
The reason it was 0.8 was that we had those formulas that changed it based on the item level, we dont have that any more and now its just +80% - so that needs to be changed, not the ApplyAffix.

Reply:
I was using spawnitem and the first item I checked was 0.5f and then the second was 0.8f and I could have sworn I was looking at an affix on the item, not the implicit.
Nonetheless I was in a rush and could have been mistaken.

Reply:
Greaves have 0.5f movement speed implicit